styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700');
+
+* {
+  box-sizing: border-box;
+  
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+span {
+  display: block;
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+  padding: 40px;
+}
+
+body {
+  background-color: #ffffff;
+  color: #434344;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 400;
+}
+
+section {
+  border-bottom: solid 2px black;
+}
+
+section:last-of-type {
+  border-bottom: none;
+}
+
+.container {
+  max-width: auto;
+  display: grid;
+  grid-template-rows: 2% 2fr 1fr 1fr 1fr 2fr 2fr 5%;
+}
+
+.nav-bar {
+  background-color: lightgoldenrodyellow
+}
+
+.nav-bar-list {
+  justify-content: center;
+  padding: 0px;
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 4%;
+  margin-left: 3%;
+  margin-right: 3%
+}
+
+.nav-bar-list li:first-child {
+  text-align: left;
+  flex: 3;
+}
+
+.nav-bar-list li:nth-child(2) {
+  flex: 1;
+}
+
+.hero {
+  background-image: url(https://i.ytimg.com/vi/bEPAbS-SRlg/maxresdefault.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-align: center;
+}
+
+h1 {
+  margin-top: 10%;
+  font-size: 3em;
+}
+
+sub {
+  font-size: 1.5rem;
+}
+
+a {
+  display: table;
+  margin-top: 50px;
+  margin-left: auto;
+  margin-right: auto;
+  border: solid 1px black;
+  border-radius: 10px;
+  padding: 10px;
+  color: black;
+  font-size: 1rem;
+}
+
+a:hover {
+  font-weight: bold;
+}
+
+blockquote {
+  width: 75%;
+  margin-top: 15%;
+  margin-left: 10%;
+  font-style: italic;
+  font-size: 1rem;
+}
+
+blockquote + p {
+  text-align: right;
+  margin-right: 10%;
+  margin-top: 15%;
+  margin-bottom: 5%;
+  font-style: italic
+}
+
+blockquote + p::before {
+  content: "-- "
+}
+
+#quote {
+  margin-left: 10px;
```

Comment:
I think using rem or em here like you do other places would help making your webpage even more accessible.